MainModule/Server/Core/Functions.lua

```diff
@@ -452,13 +452,18 @@ return function(Vargs, GetEnv)
 		end;
 
 		GetChatService = function()
+			if game:GetService('TextChatService').ChatVersion == Enum.ChatVersion.TextChatService then
+				print('TextChatService is not fully supported yet!')
+				return false
+			else
```

Comment:
A chatservice can still exist if there is a textchar. This is not necessary.

Reply:
RAHHHH

Reply:
👻